cms/src/components/groups/EditAclByGroup.spec.ts

```diff
@@ -149,4 +149,45 @@ describe("EditAclByGroup.vue", () => {
 
         expect(wrapper.html()).not.toContain('div[data-test="duplicateAcl"]');
     });
+
+    it("shows the group's ID when the group is not available to the user", async () => {
+        const wrapper = mount(EditAclByGroup, {
+            props: {
+                group: mockGroupDtoPublicContent,
+                assignedGroup: mockGroupDtoPublicContent,
+                originalGroup: mockGroupDtoPublicContent,
+                availableGroups: [
+                    mockGroupDtoPublicContent,
+                    mockGroupDtoPublicEditors,
+                    mockGroupDtoPublicUsers,
+                    mockGroupDtoSuperAdmins,
+                ],
+                disabled: true,
+            },
+        });
+
+        expect(wrapper.text()).toContain(mockGroupDtoPublicContent._id);
+        expect(wrapper.html()).not.toContain('div[data-test="duplicateAcl"]');
+    });
+
+    it("checks if acl entries are disabled when no assign permission", async () => {
+        delete accessMap.value["group-public-content"].group?.assign;
+
+        const wrapper = mount(EditAclByGroup, {
+            props: {
+                group: mockGroupDtoPublicContent,
+                assignedGroup: mockGroupDtoPublicContent,
+                originalGroup: mockGroupDtoPublicContent,
+                availableGroups: [
+                    mockGroupDtoPublicContent,
+                    mockGroupDtoPublicEditors,
+                    mockGroupDtoPublicUsers,
+                    mockGroupDtoSuperAdmins,
+                ],
+                disabled: false,
+            },
+        });
+
+        expect(wrapper.html()).not.toContain('div[data-test="duplicateAcl"]');
```

Comment:
Perhaps add a comment stating that you are checking that the group is not editable by checking that the duplicate button is not visible